### DISCUSSION TITLE
Vale a pena manter a versão em inglês? Is it worth to maintain the English Version?

### DISCUSSION THREAD

**Discussion by mateus-amarante** [mateus-amarante/awesome-robotics-brazil#17] [Q&A]
Penso que este repositório é voltado para brasileiros principalmente. Originalmente, o intuito é conectar pessoas e instituições em prol do desenvolvimento da robótica no Brasil.

Porém, a robótica é um assunto global e a maioria do conteúdo existente está em inglês. Embora o mercado brasileiro ainda não tenha tanta visibilidade hoje, no futuro esta lista pode ser de interesse de outros países. Além disso, as listas awesome também são tradicionalmente em inglês.

O principal fator negativo de manter a versão em inglês é o esforço demandado. Pequenas atualizações são OK de fazer, mas atualizações grandes são chatas e, muitas vezes, o meu sentimento é de que é esforço morto...

Por enquanto penso em manter as duas versões, porém ficaria grato de ouvir a opinião da comunidade a respeito.

**Comment by h3ct0r** [comment]
Em particular acho que é interessante sim manter uma versão em inglês, mas pode ser uma versão gerada automaticamente da versão em português. Manter as duas listas manualmente não vai ser divertido. Vou dar uma olhada em como outros resolveram esse dilema, eu acharia ótimo preparar alguma dessas ferramentas que sincronizam com o GitHub para gerar o arquivo em inglês após cada commit no master do README por exemplo, mas não sei que tão viajado é de fato fazer isso! :cartwheeling: :flying_saucer:

**Reply by mateus-amarante** [reply]
Eu havia pensado em algo nesta linha... minha ideia era armazenar as informações em um lugar (como num arquivo .yaml ou .json) e os arquivos README serem gerados automaticamente com uma GitHub Action. Porém, não sei o trabalho que isso poderia dar...

**Reply by mateus-amarante** [reply]
Outra alternativa seria usar tradutores, porém não gosto muito de depender de ferramentas de tradução. [Essas](https://github.com/marketplace/actions/translate-readme-action) [duas](https://github.com/marketplace/actions/translation-action) Github Actions podem ser úteis nesta linha.
